### PR TITLE
Remove web3

### LINKED ISSUE #183: Remove web3 dependency
As discussed here #164, we are migrating to `ethers.js` and therefore the dependency on `web3` should be removed.

Depends on #181 and should adapt library to use new implemented provider and signer.

Related to https://github.com/trustlines-network/project/issues/332

### PR DESCRIPTION
**Pull request by dakingha69**
Closes #183 

- removes web3 dependency and artifacts
- implements missing methods in `Web3Signer` with ethers

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -82,13 +82,11 @@
   },
   "dependencies": {
     "bignumber.js": "7.2.1",
-    "ethereumjs-abi": "^0.6.5",
     "ethereumjs-util": "^5.2.0",
```
Comment (berndbohmeier) [inline package.json]: Do we still need ethereumjs-utils?

Comment (dakingha69) [reply package.json]: Yes we use some ethereumjs-utils functionalities here and there. Will open an issue to replace this also with `ethers.js`.

Comment (dakingha69) [reply package.json]: https://github.com/trustlines-network/clientlib/issues/199

**Comment by dakingha69** [comment]
Only tested it locally with a button that triggers the signing. Opened an issue for this https://github.com/trustlines-network/clientlib/issues/200.

**Pull request merged**